src/Auth0.ManagementApi/Paging/PagedListConverter.cs

```diff
@@ -41,6 +41,8 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                     int limit = 0;
                     int start = 0;
                     int total = 0;
+                    string next = null;
+
```

Comment:
Can probably remove this

Reply:
Something ike this?


```c#
if (item[_collectionFieldName] != null)
{
    var collection = item[_collectionFieldName].ToObject<IList<T>>(serializer);

    if (item["next"] != null) {
        return new CheckpointPagedList<T>(collection, new CheckpointPagingInformation(item["next"].Value<string>()));
    }

    int length = 0;
    int limit = 0;
    int start = 0;
    int total = 0;

    if (item["length"] != null)
        length = item["length"].Value<int>();
    if (item["limit"] != null)
        limit = item["limit"].Value<int>();
    if (item["start"] != null)
        start = item["start"].Value<int>();
    if (item["total"] != null)
        total = item["total"].Value<int>();

    return new PagedList<T>(collection, new PagingInformation(start, limit, length, total));
}
```

Reply:
Took a stab at it, let me know what you think.